testsuite/src/test/groovy/noe/tomcat/JvmRouteTomcat7ConfiguratorIT.groovy

```diff
@@ -1,12 +1,15 @@
 package noe.tomcat
 
+import noe.common.utils.Platform
+import org.junit.Assume
 import org.junit.BeforeClass
 
 
 class JvmRouteTomcat7ConfiguratorIT extends JvmRouteTomcatConfiguratorIT {
 
   @BeforeClass
   public static void beforeClass() {
+    Assume.assumeTrue("Tomcat 7 not release for >RHEL7", new Platform().isRHEL7())
```

Comment:
Still multiple versions of RHEL limitations. Seen RHEL7+ in another change.

Reply:
Sorry, when I upgraded I only checked the ones you highlighted. Now I have searched the whole diff and edited to consistent wording (I hope)